<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="title">
     <string>Formulaire </string>
    </property>
    <widget class="QLabel" name="nom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom </string>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Modifier">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Modiifer</string>
     </property>
    </widget>
    <widget class="QLabel" name="prenom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="statut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>statut</string>
     </property>
    </widget>
    <widget class="QLabel" name="adresse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse </string>
     </property>
    </widget>
    <widget class="QComboBox" name="Statut">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Séléctionner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sous Société</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="nom_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresse_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="num_tel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CIN">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier par </string>
    </property>
   </widget>
   <widget class="QComboBox" name="TRI">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Lettres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nouvel élément</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="PDF">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>479</width>
       <height>249</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse mail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Supprimer">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>SUPPRIMER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Recherche">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LISTE DES CLIENTS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>MODIFIER</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_clients">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>291</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>saisir ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_supp">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menugg">
    <property name="title">
     <string>Gestion client</string>
    </property>
   </widget>
   <addaction name="menugg"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGestion_client">
   <property name="text">
    <string>Gestion client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
